.github/label-actions.yml

```diff
@@ -25,3 +26,14 @@ There are two types of logs to collect:
 - Run the command: **`Developer: Open Logs Folder`**
 
 - This will open the log folder locally. Please zip up this folder and attach it to the issue."
+
+# actions for Out of Scope label
+Out of Scope:
+  comment: "Thank you for opening this suggestion!  This enhancement is not planned in our
+medium-term roadmap.  The issue is being closed to reduce active issues to focus on
+enhancements that are being considered for an upcoming release.  We will review closed issues
+with the 'Out of Scope' label when doing long-term planning."
+  close: true
+
+-Out of Scope:
```

Comment:
Did you verify if this works? We use https://github.com/hramos/respond-to-issue-based-on-label, but looking at the project that it was based on it seems like special logic is needed to specifically handle the "unlabeled" event : 

https://github.com/dessant/label-actions/blob/201c706f66d13baf871b4af784bbdb7555e983b4/src/index.js#L141

which I don't see in the one we use. 

If it doesn't then it looks like we might be able to just switch over to the other action instead? `dessant/label-actions@v2` https://github.com/dessant/label-actions#examples

Reply:
Oh, I read the wrong help doc 😄.  I'll remove the "label removed" action for now, since I only added it since the doc said I could.  I don't mind clicking the Reopen button when removing the label.